### ISSUE TITLE
Intro starts off with a defense

### ISSUE THREAD

**Issue by skade** [lnwdr/php-introduction#2]
I find the intro very upfront, but also very defensive. I would cut the whole intro and replace it by an intro about PHP in itself and move the "PHP is called horrible" into a very shallow comparison to other languages.

**Comment by lnwdr** [comment]
I'm still not sure about that intro. I get the impression that man people arrive at PHP already with the notion that it's a nightmare and I want to take that off a bit. Not sure, if I have the right approach, though.

**Comment by xMartin** [comment]
I agree this sets a bad mood and while it might help motivate some people it also might demotivate and confuse others depending on where you're coming from.

Still, I think the text is well written and valuable as it is. Maybe move it somehow.
